runtime/nls/cfre/cfrerr.nls

```diff
@@ -1590,3 +1590,79 @@ J9NLS_CFR_ERR_PREVIEW_VERSION2.explanation=Classfiles with minor version 65535 m
 J9NLS_CFR_ERR_PREVIEW_VERSION2.system_action=The JVM will throw a verification or classloading related exception.
 J9NLS_CFR_ERR_PREVIEW_VERSION2.user_response=Run with the correct Java runtime version and use --enable-preview, or re-compile the code with the same version of the runtime.
 # END NON-TRANSLATABLE
+
+J9NLS_INVALID_REFERENCETYPE_DESCRIPTOR_ON_CLASS=A ReferenceType descriptor cannot be referenced by a CONSTANT_Class_info structure that represents this_class
```

Comment:
I see there are 3 ways of saying "reference type" in this file:

"ReferenceType", "referencetype" and "reference type". Better to be consistent.
Better to use "reference type" if you want it to be translated. Or use "ReferenceType" and add `#  ReferenceType is not translatable` above the message.